### DISCUSSION TITLE
Invalid copilot token: missing token

### DISCUSSION THREAD

**Discussion by eormeno** [community/community#8159] [Copilot Conversations]
Hi! I was working with copilot during about one month but yesterday copilot stop working and in the log says "Invalid copilot token: missing token". Coincidentally, yesterday I had to renew the git token. Could be this that generated the problem? Could I resolve it?

Thanks

**Comment by ILTShade** [comment]
You can sign in your github account on your browser. And click into the github setting, then find a sidebar named github copilot, select and option and save. Then restart your vs code

**Comment by lonnieezell** [comment]
I'm having the same issue. VSCode tells me it's not fully configured so I go to Copilot Settings in GitHub, select some options, Save, then restart VSCode. Always comes back with the same error that it's not configured.
